### PR TITLE
normalize comments

### PR DESCRIPTION
**Pull request by rkimballn1**
Normalizing comments to be `//[space]`

Opinions?

### CODE REVIEW

**Comment by rkimballn1** [comment]
I just used a regex in VS Code to do the search/replace

File: src/ngraph/op/argmax.hpp [hunk 1]
```diff
@@ -25,7 +25,7 @@ namespace ngraph
 {
     namespace op
     {
-        //brief Computes minimum index along a specified axis for a given tensor
+        // brief Computes minimum index along a specified axis for a given tensor
         class ArgMax : public op::util::IndexReduction
```
Comment (diyessi) [inline src/ngraph/op/argmax.hpp]: \brief

File: src/ngraph/op/argmin.hpp [hunk 2]
```diff
@@ -25,7 +25,7 @@ namespace ngraph
 {
     namespace op
     {
-        //brief Computes minimum index along a specified axis for a given tensor
+        // brief Computes minimum index along a specified axis for a given tensor
```
Comment (diyessi) [inline src/ngraph/op/argmin.hpp]: \brief

File: src/ngraph/pass/core_fusion.cpp [hunk 3]
```diff
@@ -139,7 +139,7 @@ void pass::CoreFusion::construct_sigmoid_bprop()
     auto broadcast_constant = std::make_shared<op::Broadcast>(constant, Shape{3, 4}, AxisSet{0, 1});
 
     auto add_exp = std::make_shared<op::Add>(exp_neg_input, broadcast_constant);
-    // //auto divide_1_over_exp = std::make_shared<op::Divide>(broadcast_constant, add_exp);
+    // // auto divide_1_over_exp = std::make_shared<op::Divide>(broadcast_constant, add_exp);
```
Comment (diyessi) [inline src/ngraph/pass/core_fusion.cpp]: double commented

**Pull request merged**